src/main/java/pages/HomePage.java

```diff
@@ -31,55 +32,76 @@
     @FindBy(linkText = "Edit Profile Settings")
     private WebElement lnkEditProfileSettings;
 
+    @FindBy(css = ".article-preview:nth-child(1) .ion-heart")
+    private WebElement heartIcon;
+
     @FindBy(xpath = "//a[@href='/settings']")
     private WebElement lnkSettings;
 
     public HomePage() {
         super();
     }
 
-    public boolean isVisibleImgUser(){
+    public boolean isVisibleImgUser() {
         return imgUser.isDisplayed();
     }
 
-    public void clickGlobalFeedLnk(){
+    public void clickGlobalFeedLnk() {
         clickElement(lnkGlobalFeed);
     }
 
-    public String getFirstArticle(){
+    public String getFirstArticle() {
         return getTextElement(listArticles.get(0));
     }
 
-    public void clickUserPic(){
+    public void clickUserPic() {
         clickElement(imgUser);
     }
 
-    public boolean isVisibleYourFeedLink(){
+    public boolean isVisibleYourFeedLink() {
         forceWait();
         return elementIsDisplayed(lnkYourFeed);
     }
-    public boolean isVisibleMyArticlesLink(){
+
+    public void clickYourFeedLink() {
+        clickElement(lnkYourFeed);
+    }
+
+    public boolean isVisibleMyArticlesLink() {
         forceWait();
         return elementIsDisplayed(lnkMyArticles);
     }
-    public boolean isVisibleFavoritedArticles(){
+
+    public boolean isVisibleFavoritedArticles() {
         return elementIsDisplayed(lnkFavoritedArticles);
     }
 
-    public void clickEditProfileSettingsLink(){
+    public void clickFavoritedArticlesLink() {
+        clickElement(lnkFavoritedArticles);
+    }
+
+    public void clickEditProfileSettingsLink() {
         clickElement(lnkEditProfileSettings);
     }
 
-    public void clickSettingsLink(){
+    public void clickSettingsLink() {
         clickElement(lnkSettings);
     }
 
-    public String getUserNameLink(){
+    public String getUserNameLink() {
         return imgUser.getAttribute("alt");
     }
 
-    public void waitForUserNameChange(String userName){
-        wait.waitForAttributeValue(timeOutSeconds,imgUser,"alt", userName);
+    public void waitForUserNameChange(String userName) {
+        wait.waitForAttributeValue(timeOutSeconds, imgUser, "alt", userName);
+    }
+
+    public int getAmountFavorites() {
+        return Integer.parseInt(getTextElement(heartIcon.findElement(By.xpath(".."))));
```

Comment:
with By.xpath ("..") I get the parent of the heartIcon element, since the counter is part of the parent
but the change is made to work with the element css=.article-preview:nth-child(1) .btn-sm